core/metacling/src/CMakeLists.txt

```diff
@@ -61,6 +54,14 @@ ROOT_OBJECT_LIBRARY(MetaCling
   TClingValue.cxx
 )
 
+target_include_directories(MetaCling PRIVATE ${CMAKE_SOURCE_DIR}/core/metacling/res)
```

Comment:
Can we use relative paths here? That would prevent hard-wiring the build time path